lld/ELF/Relocations.cpp

```diff
@@ -1461,6 +1493,33 @@ template <class ELFT, class RelTy> void RelocationScanner::scanOne(RelTy *&i) {
     }
   }
 
+  // if (config->emachine == EM_AARCH64 && type == R_AARCH64_AUTH_ABS64) {
```

Comment:
Why is this commented out?

Reply:
Ah, thanks, removed that. The logic was moved to `RelocationScanner::processAux`